chapters/textures.txt

```diff
@@ -839,33 +839,16 @@ sparse unbound texel is undefined:.
 If the image view has a depth/stencil format, the depth component is
 selected by the pname:aspectMask, and the operation is an code:OpImage*Dref*
 instruction, a depth comparison is performed.
-The value of the result [eq]#D# is [eq]#1.0# if the result of the compare
-operation is [eq]#true#, and [eq]#0.0# otherwise.
-The compare operation is selected by the pname:compareOp member of the
-sampler.
-
-[latexmath]
-+++++++++++++++++++
-\begin{aligned}
-D & = 1.0 &
-  \begin{cases}
-    D_{\textit{ref}} \leq D_{\textit{tex}} & \text{for LEQUAL}   \\
-    D_{\textit{ref}} \geq D_{\textit{tex}} & \text{for GEQUAL}   \\
-    D_{\textit{ref}} < D_{\textit{tex}}    & \text{for LESS}     \\
-    D_{\textit{ref}} > D_{\textit{tex}}    & \text{for GREATER}  \\
-    D_{\textit{ref}} = D_{\textit{tex}}    & \text{for EQUAL}    \\
-    D_{\textit{ref}} \neq D_{\textit{tex}} & \text{for NOTEQUAL} \\
-    \textit{true}                          & \text{for ALWAYS}   \\
-    \textit{false}                         & \text{for NEVER}
-  \end{cases} \\
-D & = 0.0 & \text{otherwise}
-\end{aligned}
-+++++++++++++++++++
-
-where [eq]#D~tex~# is the texel depth value and [eq]#D~ref~# is the
-reference value from the SPIR-V operand.
-If the image being sampled has a fixed-point format then the reference value
-is clamped to [0, 1] before the comparison operation.
+The value of the result, [eq]#D#, is [eq]#1.0# if the result of the
+comparison is [eq]#true#, and [eq]#0.0# otherwise.
```

Comment:
Would make it a separate paragraph after the following two sentences.

Defining `D` is meaningless now, or where is it invoked?

Reply:
D is used to refer to the depth component in a couple of places in this chapter, but I will tweak the sentence a bit.